plugin/src/main/java/org/jenkinsci/plugins/casc/ConfigurationAsCode.java

```diff
@@ -454,7 +486,7 @@ public Node getSingleNode() {
      * @param path base path to start (can be file or directory)
      * @return list of all paths matching pattern. Only base file itself if it is a file matching pattern
      */
-    public List<Path> configs(String path) throws ConfiguratorException {
+    public static List<Path> configs(String path) throws ConfiguratorException {
```

Comment:
Could we avoid adding static?

Reply:
I thought it is a common practise use static methods over object method, is not it?

Reply:
but there is no need for it to be static, the change is unrelated.

Reply:
I add static here because It's used by new static method `getConfigFromSources` However now I that `configs` is a public API and maybe it was not a best choice(

Reply:
there should be no reason for `getConfigFromSources` to be static either 😞